### DISCUSSION TITLE
Can't get VLC to play stream

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#376] [Q&A]
Firstly I'm new to all this stuff so please bear with me.

I believe I have the sling server running ok on my laptop linked to my ProHD box see what I get when I run sling server from my command prompt window below..................

Version : 3.08e Running on Windows-10-10.0.19044-SP0 pid= 13808
Using config file config.ini
Connection Manager Running on port 65432 with 2 max streams using URL slingbox.
BOXES [('sb2', 'Solo')]
Streamer Running:  2 config.ini Solo Solo 65432 8388608
starting up on  port 65432 8388608
Checking for slingbox at ('192.168.0.254', 5001)
('192.168.0.254', 5001) OK
Solo Using slingbox at  ('192.168.0.254', 5001)
Streamer:  Solo Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Building page for Solo
Reading Custom Remote definition from solo_remote.txt
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off

I've not used VLC for streaming before - just playing video files. What I did was start VLC, click on MEDIA, OPEN NETWORK STREAM.  I then entered http://192.168.0.10:8080/slingbox as the network URL and clicked on PLAY.

I then get an Error dialogue saying "Your input can't be opened. VLC is unable to open the MRL 'http://192.168.0.10:8080/slingbox"

Am I doing something wrong in VLC ??  If so how should I be using VLC to access the video stream ?

Thanks

**Comment by GerryDazoo** [comment]
"Connection Manager Running on port **65432** with 2 max streams using URL slingbox."
You're using the wrong port number.
http://192.168.0.10:65432/slingbox
